### PR TITLE
Add NAS KWS model (trained using dynamic augmentation)

### PR DESCRIPTION
**Pull request by alicangok**
The trained networks in this PR achieve the following accuracies for the [current version](https://github.com/MaximIntegratedAI/ai8x-training/pull/280) of our KWS20 testing set[^Note]

- v2: 92.0%
- v3: 90.8%
- NAS: 93.8% 

[^Note]: This testing set contains all 11005 test examples from the original Google Speech Commands dataset without any "background/others" class balancing, as opposed to many [benchmark settings](https://paperswithcode.com/sota/keyword-spotting-on-google-speech-commands). For our dataset, the "others" class makes up 25.7% of the testing set, making it more challenging than the aforementioned benchmarks which feature a much smaller "others" class (10%), as well as an easily identifiable "silence" class (10%).

### CODE REVIEW

**Comment by alicangok** [comment]
PR changed to draft mode, awaiting training completion for v2 & v3 models with dynamic augmentation.

File: gen-demos-max78000.sh [hunk 1]
```diff
@@ -11,6 +11,7 @@ python ai8xize.py --test-dir $TARGET --prefix cifar-100-mixed --checkpoint-file
 python ai8xize.py --test-dir $TARGET --prefix cifar-100-simplewide2x-mixed --checkpoint-file trained/ai85-cifar100-simplenetwide2x-qat-mixed-q.pth.tar --config-file networks/cifar100-simplewide2x.yaml --softmax $COMMON_ARGS --boost 2.5 "$@"
 python ai8xize.py --test-dir $TARGET --prefix cifar-100-residual --checkpoint-file trained/ai85-cifar100-residual-qat8-q.pth.tar --config-file networks/cifar100-ressimplenet.yaml --softmax $COMMON_ARGS --boost 2.5 "$@"
 python ai8xize.py --test-dir $TARGET --prefix kws20_v3 --checkpoint-file trained/ai85-kws20_v3-qat8-q.pth.tar --config-file networks/kws20-v3-hwc.yaml --softmax $COMMON_ARGS "$@"
+python ai8xize.py --test-dir $TARGET --prefix kws20_nas --checkpoint-file trained/ai85-kws20_nas-qat8-q.pth.tar --config-file networks/kws20-nas-hwc.yaml --softmax $COMMON_ARGS "$@"
```
Comment (ermanok) [inline gen-demos-max78000.sh:14]: Is it possible to rename the KWS20_v3 and KWS20_nas models as kws_light and kws? We can also remove the v1 & v2 models...

**Review by aniktash — COMMENTED** [review]
Looks good

**Comment by alicangok** [comment]
Also modified copyright texts for files generated by ai8xize.py (namely `cnn.c`, `cnn.h`, `main.c`, `Makefile`, and `softmax.c`), following @sihyung-maxim's comments in https://github.com/Analog-Devices-MSDK/msdk/pull/851 .

The PR is now marked as "Ready".

**Review by MaximGorkem — APPROVED** [review]
Looks good

File: gen-demos-max78000.sh (hunk 1 above)
Comment (ermanok) [reply gen-demos-max78000.sh:14]: That makes sense. Thanks...

**Review by rotx-maxim — APPROVED** [review]

**Pull request merged**